trieste/utils/pareto.py

```diff
@@ -106,6 +113,164 @@ def _bounds_2d(front: TensorType) -> BoundedVolumes:
 
         return BoundedVolumes(lower, upper)
 
+    @staticmethod
+    def _is_test_required(smaller: TensorType) -> TensorType:
+
+        # Tests if a point augments or dominates the Pareto set.
+        # :param smaller: a boolean tf.Tensor storing test point < Pareto front
+        # :return: True if the test point dominates or augments the Pareto front (boolean)
+
+        # if and only if the test point is at least in one dimension smaller
+        # for every point in the Pareto set
+        idx_dom_augm = tf.reduce_any(smaller, axis=1)
+        is_dom_augm = tf.reduce_all(idx_dom_augm)
+
+        return is_dom_augm
+
+    def divide_conquer_nd(self, front: TensorType, threshold: TensorType = 0) -> TensorType:
```

Comment:
any reason this is public, and if so, why is the 2D version not public?

Reply:
changed it to private